src/app/shell/personal-cabinet/provider/create-workshop/create-about-form/create-about-form.component.html

```diff
@@ -172,15 +172,14 @@
 
   <div *ngIf="provider.ownership !== ownershipType.State" fxLayout="column" fxLayoutAlign="center space-between">
     <div fxLayout="row" fxLayoutAlign="left space-between">
-      <label class="step-label">{{ 'FORMS.LABELS.AVAILABLE_PLACES' | translate }}<span
-        class="step-required">*</span></label>
+      <label class="step-label">{{ 'FORMS.LABELS.AVAILABLE_PLACES' | translate }}<span class="step-required">*</span></label>
       <i
         [ngClass]="opened ? 'material-icons status-info-icon activeInfoBtn' : 'material-icons status-info-icon inactiveInfoBtn'"
         [matMenuTriggerFor]="unlimitedInfo"
         #childTrigger="matMenuTrigger"
         (menuOpened)="opened = childTrigger.menuOpen"
         (menuClosed)="opened = childTrigger.menuOpen"
-      >info_outline</i
+        >info_outline</i
       >
```

Comment:
Leave the closing brackets at the same line. 
There are few places in this file with same issue